css/layout-css/agenda-style.upload.css

```diff
@@ -1525,18 +1525,43 @@
 .agenda-multiple-images-holder {
   display: flex;
   flex-wrap: wrap;
-  justify-content: space-around;
 }
 
 .agenda-multiple-images-item {
-  flex: 1 1 70px;
-  margin: 10px;
-  border-radius: 5px;
-  overflow: hidden;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  width: 149px;
```

Comment:
How is this making sure that it's always presented as a 2 or 3 column layout in squares if the width and height are set in absolute values?

Reply:
How can we set it in percentage?

Reply:
@tonytlwu 
To make sure that the column has 2 or 3 items per row we can add max-width to the images container in px. So, far this is the only way we have found. Is it ok ?

Reply:
@MaksymShokin I don't understand what the challenge is here. There are many methods in CSS to create a 2 or 3 column layout.

Reply:
@tonytlwu 
Columns-count property does not work properly here due to wrong ordering. Could you share other methods you have in mind? I can use css grid. But we still need to set absolute values for height and width.

Reply:
https://jsfiddle.net/tonytlwu/7r26b8cz/

Reply:
I did not get how is jsfiddle above is going to help. We have 1 array of elements that we have to sort in 2 or 3 items per row.

Reply:
@MaksymShokin My mistake. I hadn't tested with more than 3 items. See https://jsfiddle.net/tonytlwu/7r26b8cz/ again now.

Reply:
@tonytlwu 
I do not see any changes. If I add fourth item it does not go onto the next row.

Reply:
@MaksymShokin Sorry. The link was incorrect. See https://jsfiddle.net/tonytlwu/7r26b8cz/3

Reply:
@tonytlwu 
In the requirements, it was said that containers should be squares. You added width: 33.33%. How can we add the same value to the height?

Reply:
Use either one of these principles https://jsfiddle.net/tonytlwu/bL2hkcae/

Reply:
Fixed